lib/bracket-matcher-view.coffee

```diff
@@ -24,7 +25,9 @@ class BracketMatcherView
     @tagFinder = new TagFinder(@editor)
     @pairHighlighted = false
     @tagHighlighted = false
+    @commentOrStringSelector = SelectorCache.get('comment.* | string.*')
 
+    @subscriptions.add @editor.onDidTokenize(@updateMatch)
```

Comment:
hey @john-f, what was the thought behind this addition? It's not quite clear to me why it's necessary. Commenting it out and running the tests, it seems that they all pass.